react/the_react_ecosystem/fetching_data_in_react.md

```diff
@@ -253,4 +251,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- [TanStack Query documentation](https://tanstack.com/query/latest/docs/react/overview)
+- [TanStack Query documentation](https://tanstack.com/query/latest/docs/react/overview) is a library that handles all the necessary states and offers built-in support for major features, such as catching.
```

Comment:
Just want to double check, did you mean "catching' or "caching"?

Reply:
Oh my.

Reply:
The instance in the lesson overview might be correct with "catching" cc @01zulfi for confirmation on the context

Reply:
I see, I've re-read and yes, that does make sense.

Reply:
Reverted the change.

Reply:
Yup, lesson overview is "Catching"